src/pa01/snakes_and_ladders.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+import random as rd
+import numpy as np
+
+
+__author__ = "Fábio Rodrigues Pereira and Rabin Senchuri"
+__email__ = "faro@nmbu.no and rase@nmbu.no"
+
+
+def single_game(num_players):
+    """
+    Returns duration of single game.
+
+    Arguments
+    ---------
+    num_players : int
+        Number of players in the game
+
+    Returns
+    -------
+    num_moves : int
+        Number of moves the winning player needed to reach the goal
+    """
+    player = create_players(num_players)
+    has_won = False
+    winner_moves = 0
+    # Initiate a game
+    while not has_won:
+        # Initiate a round of the game
+        for p in range(num_players):
+            player[p][0] += rd.randint(1, 6)
+            player[p][1] += 1
+            # checks if the players are at a ladder
+            if player[p][0] in chutes_and_ladders():
+                player[p][0] = chutes_and_ladders()[player[p][0]]
+                player[p][1] += 1
+            # checks if some player passed the 90th square
+            elif player[p][0] > 90:
+                winner_moves = player[p][1]
+                has_won = True
+                break
+    return winner_moves
```

Comment:
The number of moves for each player is the same (even when the game stops) so you don't need to store it together with the positions. Instead, you just need calculate the number of rounds until one player wins then you can get the winner_moves